server/flaskr/database/handlers/role_data_handler.py

```
@@ -5,20 +5,20 @@
     @classmethod
     def create_role(
-        cls, name: str, canWrite=False, canDelete=False, canUpdatePermissions=False
+        cls, name: str, contribute=False, deleteContent=False, manageTeam=False
     ) -> Role | None:
         try:
             with get_db_access() as conn:
                 cur = conn.cursor()
 
-                query = f"INSERT INTO roles (name, canWrite, canDelete, canUpdatePermissions) values (%s, %s, %s, %s) RETURNING id;"
-                params = (name, canWrite, canDelete, canUpdatePermissions)
+                query = f"INSERT INTO roles (name, contribute, deleteContent, manageTeam) values (%s, %s, %s, %s) RETURNING id;"
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)